thirdeye-persistence/src/main/java/ai/startree/thirdeye/datalayer/bao/TaskManagerImpl.java

```diff
@@ -67,6 +67,8 @@ public class TaskManagerImpl implements TaskManager {
   private static final String FIND_BY_NAME_ORDER_BY_CREATE_TIME_DESC =
       " WHERE name = :name order by createTime desc limit ";
 
+  private static Timestamp taskLatencyWindowStart = new Timestamp(System.currentTimeMillis());
```

Comment:
This will be emitted from all pods i.e. coordinator, scheduler, worker and something like max(latency from all pods) will be done to get the value on which alert or any further action is decided.